ratis-test/src/test/java/org/apache/ratis/datastream/TestDataStream.java

```diff
@@ -137,42 +139,30 @@ public void testDataStream(){
   public void runTestDataStream(){
     final int bufferSize = 1024*1024;
     final int bufferNum = 10;
-    final DataStreamOutput out = client.stream();
-
-    //send request
-    final List<CompletableFuture<DataStreamReply>> futures = new ArrayList<>();
-    futures.add(sendRequest(out, 1024));
-
-    //send data
-    final int halfBufferSize = bufferSize/2;
-    int dataSize = 0;
-    for(int i = 0; i < bufferNum; i++) {
-      final int size = halfBufferSize + ThreadLocalRandom.current().nextInt(halfBufferSize);
-      final ByteBuffer bf = initBuffer(dataSize, size);
-      futures.add(out.streamAsync(bf));
-      dataSize += size;
-    }
 
-    //join all requests
-    for(CompletableFuture<DataStreamReply> f : futures) {
-      f.join();
-    }
-    Assert.assertEquals(dataSize, byteWritten);
-    shutDownSetup();
-  }
+    ByteBuffer buf = initBuffer(bufferSize);
+    Message msg = Message.valueOf(ByteString.copyFrom(buf));
```

Comment:
We don't want to use ByteString (or Message) in DataStream.  Otherwise, we won't be able to support zero buffer copying.